app/change_set_persisters/change_set_persister/preserve_resource.rb

```diff
@@ -0,0 +1,114 @@
+# frozen_string_literal: true
+
+class ChangeSetPersister
+  class PreserveResource
+    attr_reader :change_set_persister, :change_set, :post_save_resource
+
+    def initialize(change_set_persister:, change_set:, post_save_resource: nil)
+      @change_set = change_set
+      @change_set_persister = change_set_persister
+      @post_save_resource = post_save_resource
+    end
+
+    def run
+      return unless change_set.try(:preserve?)
```

Comment:
I suspect we will eventually wind up with multiple preservation policies, e.g. one for Coldline, one for S3, one for campus storage, etc. Or maybe one for Coldline, and a second policy for Coldline and S3.  Maybe this should lookup a config based on the preservation policy instead of being a binary?